apps/src/blockly/addons/contextMenu.js

```diff
@@ -0,0 +1,187 @@
+import GoogleBlockly from 'blockly/core';
+
+const registerDeletable = function() {
+  /** @type {!GoogleBlockly.ContextMenuRegistry.RegistryItem} */
```

Comment:
Can you explain this `@type` a bit? is this saying that this is not a registry item?